rules/style.js

```diff
@@ -25,9 +25,9 @@ module.exports = {
 		"eol-last": 2,
 		// Enforce naming of optionally named functions (that could remain anonymous). This helps with debugging, but open to turning this off
 		"func-names": 2,
-		// Enforce a mininmum id length of 2 characters, except i for iterating over loops
+		// Enforce a minimum id length of 2 characters, except i for iterating over loops
 		"id-length": [2, {
-			"exceptions": ["i", "_"]
+			"exceptions": ["i", "j", "k", "_"]
```

Comment:
Historically I thought `_` was used to designate an imported variable that's not used (i.e. `(_, req)` in express. I assume we have instances of this being used in our code base since this is common practice, but I'm not sure how to figure out if so and how many there are. If we remove this is there another approach? Would we just do `(, req)`?

Reply:
Cool, I'll take it out and see if any new errors are thrown. If so, I'll see how many and either refactor or add it back in.